### PR TITLE
initial food details rendering

### PR DESCRIPTION
**Pull request by kevinsa**

### CODE REVIEW

File: client/src/containers/journal/components/FoodItem.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    }
+
+    this.state = {
+      selectedServing: selectedServing
+    }
+  }
+
+  onServingChange = (event) => {
+    const servingId = event.target.value;
+    const serving = this.props.food.servings.serving.find(serving => {
```
Comment (joseph4tw) [inline client/src/containers/journal/components/FoodItem.js]: I believe you can shorten this to:
```js
const serving = this.props.food.servings.serving.find(serving => serving.serving_id === servingId);
```

File: client/src/containers/journal/components/FoodItem.js [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    })
+
+    this.setState({
+      selectedServing: serving
+    });
+  }
+
+  renderServings() {
+    if(Array.isArray(this.props.food.servings.serving)) {
+      const options = this.props.food.servings.serving.map(serving => {
```
Comment (joseph4tw) [inline client/src/containers/journal/components/FoodItem.js]: If your arrow function has only one line of code you can omit the curly braces and return statement.

File: client/src/containers/journal/components/FoodItem.js [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+            </tr>
+          </tbody>
+        </MacrosTable>
+        </div>
+      </FoodItemContainer>
+    );
+  }
+}
+
+export default FoodItem;
```
Comment (joseph4tw) [inline client/src/containers/journal/components/FoodItem.js]: Just a heads up, if you use the airbnb linter, it will complain when there is no newline at the end of each `js` doc.

File: client/src/containers/journal/components/FoodSearch.js [hunk 4]
```diff
@@ -1,17 +1,20 @@
 import React from 'react';
 import FoodSearchForm from './FoodSearchForm';
 import FoodSearchResults from './FoodSearchResults';
+import FoodItemList from './FoodItemList';
 import Pager from '../../../components/Pager';
-import { SearchFoods } from '../../../services/foods';
+import { SearchFoods, GetFood } from '../../../services/foods';
```
Comment (joseph4tw) [inline client/src/containers/journal/components/FoodSearch.js]: If I remember correctly, the airbnb linter wants multiple imports to be sorted alphanumerically.

**Comment by kevinsa** [comment]
[Food Details: Implement display of food details when a food item is selected from the search results](https://trello.com/c/KtnTiFfI/26-food-details-implement-display-of-food-details-when-a-food-item-is-selected-from-the-search-results)

File: client/src/containers/journal/components/FoodItem.js (hunk 1 above)
Comment (kevinsa) [reply client/src/containers/journal/components/FoodItem.js]: Yes, that works since it's a single return statement

File: client/src/containers/journal/components/FoodItem.js (hunk 2 above)
Comment (kevinsa) [reply client/src/containers/journal/components/FoodItem.js]: Yes, that also works, no need for braces here either

File: client/src/containers/journal/components/FoodSearch.js (hunk 4 above)
Comment (kevinsa) [reply client/src/containers/journal/components/FoodSearch.js]: The individual imports within {} or all import statements in the file (top to bottom)?

Comment (joseph4tw) [reply client/src/containers/journal/components/FoodSearch.js]: Good question. Definitely the individual imports within the curly braces. I'm not sure how we should handle each import line, though.
